apps/web/src/lib/functions/util.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * @license BSD-3-Clause
+ * Copyright (c) 2022, ッツ Reader Authors
+ * All rights reserved.
+ */
+
+export function generateUUID() {
+  const array = new Uint32Array(16);
+  window.crypto.getRandomValues(array);
```

Comment:
Think I always take window as a parameter, instead of global reference, please update it to take in (window: Window)